ClientApp/package.json

```diff
@@ -26,7 +26,6 @@
     "ajv": "^6.9.1",
     "bootstrap": "^4.6.0",
     "cross-env": "^5.2.0",
-    "eslint": "^6.8.0",
```

Comment:
this seems a little weird to me and the change to package-lock.json, could we revert any changes to these 2 files as I don't think we need them? Feel free to grab me if you want any help :)

Reply:
sorted